src/pipelines/rust_app.rs

```diff
@@ -57,6 +123,7 @@ impl RustApp {
             })
             .unwrap_or_else(|| html_dir.join("Cargo.toml"));
         let bin = el.attr("data-bin").map(|val| val.to_string());
+        let wasm_opt = el.attr("wasm-opt").map(|val| val.parse()).transpose()?.unwrap_or_default();
```

Comment:
After rebasing onto master, this will have to be updated as a different mechanism is being used to pass along attributes.